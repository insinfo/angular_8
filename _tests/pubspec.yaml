name: _tests
description: Tests for AngularDart.
publish_to: none

environment:
  sdk: ">=3.0.0 <4.0.0"

dependencies:
  build_resolvers: ^2.1.0
  build_test: 2.2.1
  collection: 1.18.0  
  ngdart: #^8.0.0-dev.1
     path:
      ../
  

  test: ^1.22.2

dev_dependencies:
  analyzer: 5.13.0
  build_runner: 2.4.6
  build_web_compilers: 4.0.6
  csslib: 0.17.2
  lints: ^2.0.1
  meta: ^1.8.0
  mockito: ^5.3.2
  package_config: ^2.1.0
  source_span: ^1.9.1
  term_glyph: ^1.2.1


