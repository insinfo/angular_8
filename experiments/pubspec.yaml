name: example

environment:
  sdk: ">=3.1.0 <4.0.0"

dependencies:
  analyzer: ^6.3.0 #5.12.0
  build: ^2.4.1
  built_collection: ^5.1.1
  #build_modules: '>=4.0.7 < 6.0.0'
  built_value: ^8.7.0
  collection: ^1.18.0
  intl: ^0.18.1
  js: ^0.6.7
  meta: ^1.11.0  
  stream_transform: ^2.1.0
  source_gen: ^1.4.0 #1.2.6
  source_span: ^1.10.0
  mockito: any
  charcode: ^1.3.1 #updated
  string_scanner: ^1.2.0 #updated
  args: ^2.4.2
  code_builder: ^4.5.0
  csslib: ^1.0.0
  dart_style: ^2.3.2
  logging: ^1.2.0
  path: ^1.8.3
  package_config: ^2.1.0
  ngdart:
    path:
      ../
  
dev_dependencies:
  async: ^2.10.0
  build_runner: ^2.4.6
  build_test: ^2.2.1
  build_web_compilers: ^4.0.6
  lints: ^2.1.0
  #mockito: 5.4.2  
  test: ^1.22.2
