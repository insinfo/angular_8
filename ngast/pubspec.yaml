name: ngast
description: Parser and utilities for AngularDart templates
version: 3.0.0-dev.1
repository: https://github.com/angulardart-community/angular

environment:
  sdk: ">=3.0.0 <4.0.0"

dependencies:
  charcode: ^1.3.1 #updated
  collection: ^1.18.0 #updated
  meta: 1.11.0 #updated
  source_span: ^1.10.0 #updated
  string_scanner: ^1.2.0 #updated

dev_dependencies:
  lints: ^2.0.1
  path: ^1.8.3
  #test: ^1.24.9 #updated 
  test: ^1.22.2
