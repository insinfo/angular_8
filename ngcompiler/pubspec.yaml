name: ngcompiler
description: >
  The AngularDart compiler.
  It compiles your component files written in Dart into templates so that you
  don't have to worry about all the complicated hassle.
version: 3.0.0-dev.2
repository: https://github.com/angulardart-community/angular

environment:
  sdk: ">=3.0.0 <4.0.0"

dependencies:
  analyzer: 5.13.0 #5.12.0
  args: 2.3.2
  build: 2.4.0 #
  build_test: 2.2.1 #updated
  code_builder: 4.5.0
  collection: 1.18.0
  csslib: 0.17.2
  dart_style: 2.2.4 #
  logging: 1.1.0
  meta: 1.11.0 #updated
  ngast: #^3.0.0-dev.0
    path:
      ../ngast
  ngdart: #^8.0.0-dev.1
     path:
      ../ngdart
  package_config: 2.1.0
  path: 1.8.3
  source_gen: 1.4.0 #1.2.6
  source_span: 1.10.0

dev_dependencies:
  lints: ^2.0.1
  test: ^1.22.2
