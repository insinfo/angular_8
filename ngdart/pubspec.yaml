name: ngdart
description: >-
  Fast and productive web framework by Google.
  Separate, but similar to the JavaScript framework with a similar name.
version: 8.0.0-dev.3
repository: https://github.com/angulardart-community/angular

environment:
  sdk: ">=3.0.0 <4.0.0"

dependencies:
  analyzer: 5.13.0 #5.12.0
  build: ^2.4.0
  built_collection: ^5.1.1
  build_modules: '>=4.0.7 < 6.0.0'
  built_value: ^8.4.3
  collection: ^1.18.0
  intl: ^0.18.0
  js: ^0.6.5
  meta: ^1.11.0
  ngast: #^3.0.0-dev.0
    path:
      ../ngast
  ngcompiler: #^3.0.0-dev.0
    path:
      ../ngcompiler      
  stream_transform: ^2.1.0

dev_dependencies:
  lints: ^2.0.1
