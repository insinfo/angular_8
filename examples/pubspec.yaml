name: example

environment:
  sdk: ">=3.0.0 <4.0.0"

dependencies:    
  ngdart:
    path: ../
 


dev_dependencies: 
  build_runner: 2.4.6 
  build_web_compilers: 4.0.6
  #lints: any
