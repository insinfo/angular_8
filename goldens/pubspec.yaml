name: goldens
description: Goldens for AngularDart.
publish_to: none

environment:
  sdk: ">=3.0.0 <4.0.0"

dependencies:
  js: ^0.6.5
  ngdart: #^8.0.0-dev.1
    path:
      ../ngdart


dev_dependencies:
  build_runner: 2.4.6 
  build_web_compilers: 4.0.6
  dart_style: ^2.2.4
  glob: ^2.1.1
  lints: ^2.0.1
  path: ^1.8.3
